src/pages/admin/student/edit/index.ts

```diff
@@ -0,0 +1,5 @@
+import { EditStudentPageContainer } from './pageContainer';
```

Comment:
You get advantage of it when you import out of  the folder. It's a barrel, if you want to find out more information about it.